infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -53,6 +63,17 @@ _If using environment variables_
 
 `terraform plan` to see the plan generated by terraform for the specified deployment configuration. If you are satisfied with the plan, run `terraform apply` to apply the configuration.
 
+You will need to run `terraform plan` and `terraform apply` from two dirrent directories. First, run the commands from the `../container/` directory to set up the Azure Container Registry. Then, run the commands from the `sandbox/` (current) directory to set up the rest of the sandbox.
```

Comment:
Just realize that we need to run terraform twice. Is it possible to make it one step?

Reply:
See my slack message and my [comment](https://github.com/linkedin/openhouse/pull/139#issuecomment-2248820066). As of now I don't see a simple way.

One alternative is to write a separate script that combines them. What do you think?

Reply:
As synced offline, it's ok to have two steps.